services/updateUser.service.js

```diff
@@ -0,0 +1,25 @@
+const User = require("../models/user.model");
+const { NotFoundError } = require("../helpers/errorHandlers");
+
+exports.updateUserService = async (body) => {
+  try {
+    await User.updateOne(
+      { email: body.email },
+      {
+        $set: body,
+      }
+    );
+    return res.status(200).json("updatedUser");
+  } catch (err) {
+    return err;
+  }
```

Comment:
service should return with value not res.status etc...
you also dont need try catch here (should look like deleteUser)